core/gate_hooks/track.cc

```diff
@@ -44,33 +44,32 @@ const GateHookCommands Track::cmds = {{"reset", "EmptyArg",
 Track::Track()
     : bess::GateHook(Track::kName, Track::kPriority),
       track_bytes_(),
-      cnt_(),
-      pkts_(),
-      bytes_() {}
+      worker_stats_() {}
```

Comment:
My C++ knowledge is not enough to understand why this is initialized to 0, but I tested it and it works well

Reply:
This is something like `int a{};`, which will be initialized to zero. If a type doesn't have anything but default constructor, `a()` or `a{}` will trigger value initialization (i.e., zeroed), while simply `a` incurs default initialization (i.e., doing nothing for `int` or `TrackStats`).